chain/chain/src/chain.rs

```diff
@@ -4279,19 +4284,19 @@ impl<'a> ChainUpdate<'a> {
         Ok(())
     }
 
-    /// Runs the block processing, including validation and finding a place for the new block in the chain.
-    /// Returns new head if chain head updated, as well as a boolean indicating if we need to start
-    ///    fetching state for the next epoch.
-    fn process_block(
+    fn preprocess_block(
```

Comment:
let's add a description on what preprocessing does (and does NOT do).

(for example - that here we don't do any state mutations etc etc)